pkgs/desktops/xfce/mkXfceDerivation.nix

```diff
@@ -27,6 +27,8 @@ let
     enableParallelBuilding = true;
     outputs = [ "out" "dev" ];
 
+    pos = builtins.unsafeGetAttrPos "pname" args;
```

Comment:
I believe my patch worked for me because it was in `meta`.

Reply:
It had no effect for me inside of `meta`. It only worked for me outside of `meta`.